README.md

```diff
@@ -26,15 +26,15 @@ To build everything:
 ```
 
 The output are in these folders:
-- `iou-contract/build/libs/iou-contract-1.0.jar`
-- `iou-flow/build/libs/iou-flow-1.0.jar`
+- `contracts/build/libs/contracts-1.0.jar`
+- `workflows/build/libs/workflows-1.0.jar`
```

Comment:
we should stick to `flows` instead of `workflow` for project and folder names, as `flows` is the official name in Corda lingo. `workflow` on the other hand can be confused with https://docs.corda.net/docs/cenm/1.2/workflow.html

Reply:
Saw them using workflows at bunch of places in their examples. Can change it to flows.